integration-cli/docker_api_containers_test.go

```diff
@@ -552,7 +552,7 @@ func (s *DockerSuite) TestContainerApiCommit(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-	if cmd != "{[/bin/sh -c touch /test]}" {
```

Comment:
While I agree with the new format (no {}'s) do you know why they appeared there in the first place?